### PR TITLE
Docs: Add more detail to video instructions

### PR DESCRIPTION
**Pull request by mjseaman**
Add details on why enablement videos are valuable, how to create them, and what happens with them afterward.

### CODE REVIEW

File: docs/sources/contribute/release-notes/index.md [hunk 1]
```diff
@@ -128,7 +128,11 @@ When you’re ready to add a What’s new entry, complete the following steps:
        <tr>
          <th scope="row">ENABLEMENT VIDEO (OPTIONAL)</th>
          <td>Link to the video used for enablement.</td>
-         <td></td>
+         <td>
+           <p>Enablement videos are perhaps the fastest, most engaging tool for employees and users to learn about your feature. Use them for maximum engagement.</p>
+           <p>Follow these instructions to create and upload a video: <a href="https://docs.google.com/document/d/1nCiG62FxJ9J_qLTnlzNopSsT-VEAlxCUCsrAEpWL68U/edit#heading=h.fierz9i4q8ft">Enablement video instructions</a>.</p>
+           <p>When you upload an enablement video, our Content team will be notified and, if applicable, edit it for the public and upload it to YouTube to concide with your feature going live. They need a few weeks' lead time for this.</p>
```
Comment (mjseaman) [inline docs/sources/contribute/release-notes/index.md:134]:
```suggestion
           <p>When you upload an enablement video, the Content team will be notified and, if applicable, edit it for the public and upload it to YouTube to concide with your feature going live. They need a few weeks' lead time for this.</p>
```

**Review by jdbaldry — COMMENTED** [review]
Looks reasonable to me. I realized that I forgot to update the CODEOWNERs for this file so I'll add @imatwawana manually so that they are aware

File: docs/sources/contribute/release-notes/index.md [hunk 2]
```diff
@@ -128,7 +128,11 @@ When you’re ready to add a What’s new entry, complete the following steps:
        <tr>
          <th scope="row">ENABLEMENT VIDEO (OPTIONAL)</th>
          <td>Link to the video used for enablement.</td>
-         <td></td>
+         <td>
+           <p>Enablement videos are perhaps the fastest, most engaging tool for employees and users to learn about your feature. Use them for maximum engagement.</p>
+           <p>Follow these instructions to create and upload a video: <a href="https://docs.google.com/document/d/1nCiG62FxJ9J_qLTnlzNopSsT-VEAlxCUCsrAEpWL68U/edit#heading=h.fierz9i4q8ft">Enablement video instructions</a>.</p>
+           <p>When you upload an enablement video, the Content team receives a notification, edits it for the public, and uploads it to YouTube to coincide with your feature's release. They need a few weeks' lead time for this.</p>
```
Comment (imatwawana) [inline docs/sources/contribute/release-notes/index.md:134]: My only nit is that "a few weeks" is kinda squishy. I'd say let's suggest a hard minimum time frame here.

**Comment by imatwawana** [comment]
> @imatwawana Do you want your nit resolved in this PR or should we get this in?

It can wait; go ahead and get this done.

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**